src/accumulator.jl

```diff
@@ -12,6 +12,14 @@ counter(T::Type) = Accumulator(T,Int)
 Accumulator{T,V<:Number}(dct::Dict{T,V}) = Accumulator{T,V}(copy(dct))
 counter{T}(dct::Dict{T,Int}) = Accumulator{T,Int}(copy(dct))
 
+"""
+    counter{T}(seq::AbstractArray{T})
+
+### Args:
+* `seq`: An AbstractArray
+
+`counter()` function returns an `Accumulator` object with the elements from `seq` assembled into. 
```

Comment:
Do not repeat function name. Also "assembled into" sounds weird to me: maybe just "containing the elements..."?